client/src/pages/Profile.js

```diff
@@ -5,111 +5,144 @@ import Typography from '@material-ui/core/Typography';
 import Button from '@material-ui/core/Button';
 import Grid from '@material-ui/core/Grid';
 import Paper from '@material-ui/core/Paper';
+import Divider from '@material-ui/core/Divider';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import RecipeCard from './RecipeCard';
 
-const useStyles = makeStyles(({
+import sushi1 from '../images/sushi1.png';
+import chef from '../images/chef.png';
+
+const useStyles = makeStyles(theme => ({
   root: {
-    height: "100vh",
-    background: '#F8F8FF'
+    height: '100vh',
+    flex: 1
   },
   sidebar: {
-    background: '#FFFFFF',
-    color: 'white',
-    height: "100vh",
     width: 400,
-    margin: '0px 0px 0px 0px',
-    textAlign: 'center',
-    float: 'left',
+    height: '100%',
+    background: '#FFFFFF'
+  },
+  sidebarimg : {
+    height: '45%',
+    backgroundImage: `url(${sushi1})`,
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center top",
+    backgroundSize: "100% 80%",
+
   },
-  profile_info: {
-    height: "60vh",
+  chefinfo_one: {
+    height: '10%',
   },
-  profile_bgimg: {
-    height: 100,
+  chefinfo_two: {
+    height: '25%',
```

Comment:
we can probably have better names than `chefinfo_one` and `chefinfo_two`.
Something like `chefHeader`, `chefBio` maybe